crowbar_framework/app/models/neutron_service.rb

```diff
@@ -277,6 +277,9 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
 
     network_nodes.each do |n|
       net_svc.allocate_ip "default", "public", "host",n
+      role.default_attributes["neutron"]["additional_external_networks"].each do |extnet|
+        net_svc.enable_interface "default", extnet, n
```

Comment:
Hrm... Should we add a validation check that each network listed in `additional_external_networks` actually is defined in the network barclamp?

(And I guess we want to exclude some networks, like admin, public, os_sdn, etc.)